datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/codeowners.py

```diff
@@ -0,0 +1,72 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+
+import click
+
+from ...utils import get_codeowners, get_valid_integrations
+from ..console import CONTEXT_SETTINGS, echo_failure, echo_success
+
+DIRECTORY_REGEX = re.compile(r"\/(.*)\/$")
+
+
+IGNORE_TILES = {
+    'bluematador',
+    'bonsai',
+    'buddy',
+    'launchdarkly',
+    'concourse_ci',
+    'auth0',
+    'gremlin',
+    'lacework',
+    'rigor',
+    'rookout',
+    'squadcast',
+}
+
+
+def create_codeowners_map():
+    """Creates a mapping of integrations to codeowners entries"""
+    codeowners = get_codeowners()
+    all_integrations = sorted(get_valid_integrations())
+    codeowner_map = dict.fromkeys(all_integrations)
+
+    # each valid entry looks something like:
+    # /containerd/                              @DataDog/container-integrations @DataDog/agent-integrations
+    for entry in codeowners:
+        parts = [part for part in entry.split(" ") if part != "" and part != "\t"]
```

Comment:
```suggestion
        parts = [part for part in entry.split() if part]
```

Is there a reason you can't just do a default split here?

Reply:
`parts` is sometimes empty if I do that, but I agree it is cleaner and makes more sense to do that and then check if parts is not empty later on